save-relation.js

```diff
@@ -125,9 +125,10 @@ module.exports = (Model, options) => {
     relationName,
     relatedObjects
   }) {
+    let includeRelation = instance[relationName]() ? {} : {include: relationName }
```

Comment:
@Tonifebrer is there any reason why you used let instead of const?

Reply:
Fixed